index.html

```diff
@@ -4,13 +4,27 @@
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<title>Simulateur de coût d'embauche</title>
+		<style>
+			body {
+				font-family: 'Helvetica Neue', Arial, Helvetica, sans-serif;
+				line-height: 2.5;
+			}
+
+			#widget {
```

Comment:
Why use a wrapper? Simply target `.SGMAPembauche` to give best practice advice to clients.